cmd/lvldb2elastic/config/configParser.go

```diff
@@ -0,0 +1,20 @@
+package config
```

Comment:
why lvldb in the name? We use storers. So, might change storer2elastic something

Reply:
done